doc/source/complex_composites/index.rst

```diff
@@ -0,0 +1,100 @@
+Complex Composites
+==================
+
+Many composites in |project| take advantage of various corrections or
+adjustments to produce the best looking imagery possible. The below
+sections describe the corrections used in |project| and also describe
+a few key composites provided in |project| and which of these
+corrections they use.
+
+.. _sunz_correction:
+
+Solar Zenith Angle Modification
+-------------------------------
+
+Visible channel data is not truly a "reflectance" until it has been adjusted
+for the amount of sun reflecting off of the atmosphere. While this is a very
+common change when working with numeric reflectance data, some RGB recipes
+may not include it.
+
+.. _psp_rayleigh_correction:
+
+Rayleigh Scattering Correction - Pyspectral
+-------------------------------------------
+
+Due to atmospheric molecules, some visible channels are scattered obscuring
```

Comment:
Due to the size of molecules that make up our atmosphere, some visible channel light is preferentially scattered more than others, especially at larger viewing angles.